packages/berry-fslib/sources/NodeFS.ts

```diff
@@ -1,8 +1,13 @@
 import fs, {Stats}                                         from 'fs';
+import {win32}                                             from 'path';
 
 import {CreateReadStreamOptions, CreateWriteStreamOptions} from './FakeFS';
 import {FakeFS, WriteFileOptions}                          from './FakeFS';
 
+
+const PORTABLE_PATH_PREFIX = `/mnt/`;
+const PORTABLE_PREFIX_REGEXP = /^\/mnt\/[a-z]\//;
```

Comment:
`C:\` would translate to `/mnt/c` (it could also be represented by `/mnt/c/`, but `posix.normalize` and similar functions would trim the trailing slash). I don't see right now why someone would use `C:\` as configuration for anything, but I think it's likely enough that it's worth handling that case.

Reply:
Ok, It make sense! (didn't know about `posix.normalize`)
Anyway, I replaced it by your RegExp so it's not a problem anymore 👌